### PR TITLE
Add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by penguineer**
About contributing to our project.

Work towards https://github.com/FreifunkMD/FFMD-Orga/issues/38

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+* If you are from the Magdeburg/Germany area, visit our [meetings](http://www.netz39.de/events/) "Freifunk Bastelabend" and "Freifunk Abstimmungstreffen" to talk about the roadmap and current development efforts.
+* Contact us via [E-Mail or Jabber](http://www.netz39.de/verein/kommunikation/)
+* If your contribution is not specific to the Freifunk Magdeburg community, please have a look at the respective project, e.g. [gluon](https://github.com/freifunk-gluon/gluon).
+
+To contribute, please follow these steps:
+
+1. Create a branch based off `master` with a meaningful name.
+2. When your contribution is ready, create a pull request. If you would like comments on the way, prepend "WIP: " to your PR title.
+3. If you contribution should appear in the Changelog, please include a distinct commit adding the respective Changelog entry.
+4. We will review the code and there will be comments. Please be prepared for a dialogue. We welcome all contributions, but we also care for quality and making as few errors as possible, so we may request changes or clarifications. Bear with us so that we can come to a nice result.
```
Comment (christf) [inline CONTRIBUTING.md]: Suggestion: ".. and we also care"...

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+* If you are from the Magdeburg/Germany area, visit our [meetings](http://www.netz39.de/events/) "Freifunk Bastelabend" and "Freifunk Abstimmungstreffen" to talk about the roadmap and current development efforts.
+* Contact us via [E-Mail or Jabber](http://www.netz39.de/verein/kommunikation/)
+* If your contribution is not specific to the Freifunk Magdeburg community, please have a look at the respective project, e.g. [gluon](https://github.com/freifunk-gluon/gluon).
+
+To contribute, please follow these steps:
+
+1. Create a branch based off `master` with a meaningful name.
+2. When your contribution is ready, create a pull request. If you would like comments on the way, prepend "WIP: " to your PR title.
+3. If you contribution should appear in the Changelog, please include a distinct commit adding the respective Changelog entry.
```
Comment (LeSpocky) [inline CONTRIBUTING.md]:
```suggestion
3. If your contribution should appear in the Changelog, please include a distinct commit adding the respective Changelog entry.
```

Comment (penguineer) [reply CONTRIBUTING.md]: Ah, danke. Habe ich korrigiert. :)

File: CONTRIBUTING.md (hunk 1 above)
Comment (penguineer) [reply CONTRIBUTING.md]: "We welcome all contributions and we also care for quality and making as few errors as possible" ?

**Comment by LeSpocky** [comment]
> What about mentioning the FFMD XMPP MUC instead of or in addition to the general Netz39 contact address page?

^^

**Comment by penguineer** [comment]
> > What about mentioning the FFMD XMPP MUC instead of or in addition to the general Netz39 contact address page?

Reduced redundancy: We will forget to change contact information here, e.g. when the Jabber Domain will finaly be just n39.eu ;)

However, I used the wrong link, should be rather this one: https://md.freifunk.net/kontakt/
(Also, out contact page needs an update.)

File: CONTRIBUTING.md (hunk 1 above)
Comment (penguineer) [reply CONTRIBUTING.md]: @christf Vorschlag: https://github.com/FreifunkMD/site-ffmd/pull/132/commits/b34cd8459a855aecf8b268a56d4d15531c565eab

**Comment by penguineer** [comment]
@christf Wenn Du das so okay findest, kannst Du direkt mergen.

**Pull request merged**